### PR TITLE
Add CSV source

### PR DESCRIPTION
**Pull request by hvelarde**

### CODE REVIEW

File: transmogrify/wordpress/blueprints/relatecontent.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        # Get the string with URLs from related content
+        pinged = item.get('_pinged', '')
+        if pinged == '':
+            return  # No related content
+
+        # The URL is formated with multiple URLs together without
+        # separator.  To break it into a list, I need to split on
+        # http and reconstruct the url
+        # TODO: handle HTTPS scheme
+        related_urls = ['http{0}'.format(url.rstrip('/'))
```
Comment (hvelarde) [inline transmogrify/wordpress/blueprints/relatecontent.py]: @rodfersou you have to fix here

**Comment by gforcada** [comment]
Squashing some commits together would have been nice.

**Comment by hvelarde** [comment]
yes, but it was getting old and I had no time to review it.

**Comment by hvelarde** [comment]
@gforcada reverted and fixed in #4 just to make you happy ;-)

**Pull request merged**